src/styles/index.scss

```diff
@@ -1,3 +1,10 @@
-body {
+@import './utils/variable';
```

Comment:
Ensure that the file './utils/variable.scss' exists and contains the necessary variable definitions, such as `$color_text--dark` and `$color_text--light`, to avoid compilation errors.